client/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+
+version: "3"
+services:
+  pyrog_client:
+    image: arkhn/pyrog-client:latest
+    container_name: pyrog-client
+    build:
+      context: .
+    ports:
+      - $PORT:$PORT
+    env_file: .env
```

Comment:
I don't understand how this works: the `.env` file points to `localhost` for the backend url, how does this work in docker ?

Reply:
we should probably have both a `.env` and a `.env.docker` (same as in the server) ?

Reply:
:+1: 
I don't really know why this worked indeed^^